.strigo/config.yml

```diff
@@ -0,0 +1,7 @@
+# Classes exercises configuration
+---
+classes:
+  - class_id: <your_class_id>
+    exercises:
+      - file: "../hello-world/hello.md"
```

Comment:
Um.. how about adding a comment that `file` could also be an absolute path?

Reply:
Good point. Done.